src/main/java/net/rptools/maptool/client/events/TokenHoverExit.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * This software Copyright by the RPTools.net development team, and
+ * licensed under the Affero GPL Version 3 or, at your option, any later
+ * version.
+ *
+ * MapTool Source Code is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License * along with this source Code.  If not, please visit
+ * <http://www.gnu.org/licenses/> and specifically the Affero license
+ * text at <http://www.gnu.org/licenses/agpl.html>.
+ */
+package net.rptools.maptool.client.events;
+
+import net.rptools.maptool.model.Zone;
+
+/**
+ * Event fired when the mouse exits a token.
+ *
+ * @param zone the zone for the event.
+ */
+public record TokenHoverExit(Zone zone) {}
```

Comment:
Maybe we should have the same members as TokenHoverEnter?

Reply:
Added information to match TokenHoverEnter event.